platform/ios/PolyPodApp/PodApi/messagePort.js

```diff
@@ -2,7 +2,7 @@ window.addEventListener("message", receiveMessage, false);
 
 function receiveMessage({ data }) {
     if (data.command === "log") {
-        webkit.messageHandlers[data.command].postMessage(data);
+        webkit.messageHandlers[data.command].postMessage(data.text);
```

Comment:
Is that a related change?

Reply:
Well, handling messages is within the scope. I'm just logging here the actual payload, not the payload together with the command.

Reply:
Did you test this change though? I don't see how it could work, I just checked the Swift code and it expects an object with a `text` property here - which is all it logs. AFAIK you usually don't pass primitives to `postMessage`,  but always an object, even if it contains only one property. May just be convention though, not enforced, but I'd still prefer not to change it based on what I know at the moment.